evap/rewards/tools.py

```diff
@@ -88,8 +97,18 @@ def grant_reward_points_after_evaluate(sender, **kwargs):
     request = kwargs['request']
     semester = kwargs['semester']
 
-    if grant_reward_points(request.user, semester):
-        messages.success(request, _("You just have earned reward points for this semester because you evaluated all your courses. Thank you very much!"))
+    result = grant_reward_points(request.user, semester)
+    if result:
+        granted, progress = result
+        message = ngettext("You just earned {count} reward point for this semester.",
+                           "You just earned {count} reward points for this semester.", granted).format(count=granted)
+
+        if progress >= 1.0:
+            message += " " + _("Thank you very much for evaluating all your courses.")
+        elif Course.objects.filter(participants=request.user, semester=semester, is_required_for_reward=True, state="in_evaluation").exclude(voters=request.user).exists():
+            message += " " + _("Please continue evaluating your courses.")
```

Comment:
`"We're looking forward to receiving feedback for your other courses as well."` sounds nicer :)